awx/ui_next/src/screens/Organization/OrganizationList/OrganizationList.jsx

```diff
@@ -114,118 +50,141 @@ class OrganizationsList extends Component {
         {
           data: { actions },
         },
-      ] = await promises;
-      this.setState({
-        actions,
-        itemCount: count,
-        organizations: results,
-        selected: [],
-      });
-    } catch (err) {
-      this.setState({ contentError: err });
+      ] = await Promise.all([
```

Comment:
I like how you merged the fetches into a Promise.all... makes managing `hasContentLoading` much cleaner